README.md

```diff
@@ -1 +1,23 @@
 # react-cli
+
+This repo is an exploration for making the tooling for React that provides the best developer experience.
+
+
+## Getting started should be dead simple
+
+Something like `npm install -g react-cli` then `react init cool-project`.
+
+## Gradual integration
+
+The initial setup should be super slim, but then you can add more to it. When you start to want to test your app, you can do `react add test`, if you want to use flow then `react add flow`, wanna add crash reporting then `react add sentry`...
```

Comment:
I'm agree with @tikotzky. `npm install redux --save` (as usual) will reduce complexity for a newbies.

I see a similar approach here with our `rnpm link` for react-native: you install a desired package by `npm` and then assemble it to the project using special command `rnpm link` in our case (or you can link only one dependency via `rnpm link name-of-dep`). Probably, it'll also work here: 

1. Install package you need (redux for instance)
2. Run `react link redux` or `react link` for linking all installed deps
3. Profit

What do you think, @vjeux ?